winml/lib/Api.Image/DeviceHelpers.cpp

```diff
@@ -12,8 +12,7 @@
 #include "CommonDeviceHelpers.h"
 #include "LearningModelDevice.h"
 
-namespace DeviceHelpers {
-
+using namespace _winml;
```

Comment:
why do you have the "using" here and then several instances of explicitly using it later? can you do one or the other instead of both?

Reply:
removed